### ISSUE TITLE
Package.loadSettings is deprecated.

### ISSUE THREAD

**Issue by alex** [jacobbednarz/atom-language-ini#9]
Store package settings files in the `settings/` directory instead of `scoped-properties/`
```
Package.loadSettings (/Applications/Atom.app/Contents/Resources/app/src/package.js:565:9)
<unknown> (/Applications/Atom.app/Contents/Resources/app/src/package.js:187:43)
```

**Comment by jacobbednarz** [comment]
Heya,
This should have been fixed in https://github.com/jacobbednarz/atom-language-ini/commit/f8f59b44a8f4f6062cdb28190688bb764a817a76 and released via 1.10.0

Can you try updating to the latest package and let me know if you still see this deprecation warning?

**Comment by alex** [comment]
Indeed, looks to be fixed. Thanks! :sparkles:
